toolkits/global/packages/global-ratings-survey/scss/50-components/_ratings-survey.scss

```diff
@@ -1,10 +1,17 @@
 .c-ratings-survey {
 	padding: $global-ratings-survey-container-padding;
+	margin: 0 (0 - $global-ratings-survey-container-padding);
```

Comment:
@benjclark  Could you explain why a negative margin was the solution here? Nothing wrong with negative margins _per se_ but I'm wondering why you needed to do this here.

Also, I'm a bit confused by the syntax. I think I'd write it as

```
margin: 0 calc(0 - #{$global-ratings-survey-container-padding});
```

But then again, I'm not the biggest Sass person.

Reply:
Thanks Heydon. 

I was using negative margin to eek out as much horizontal space as possible to allow for nicer amounts of spacing between icons. 

But looking at this again with fresh eyes I think it's a bad idea in general that would have some overflow issues when the component is placed in a location where there isn't any page section padding to overflow into. So I'm scrapping the idea entirely and will just reduce the gap between icons a little bit to ensure the design is ok and is not wrapping on devices such as iPhone 5 that have only 320px vw to play with.